.github/workflows/test.yml

```diff
@@ -92,3 +92,38 @@ jobs:
       - name: Generate code coverage
         run: |
           uv run nox --session "coverage"
+
+  docker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v3
+
+      - name: Build Docker image
+        uses: docker/build-push-action@v6
+        with:
+          context: .
+          file: .dockerfiles/Dockerfile
+          load: true
+          tags: django-email-relay-test:latest
+          push: false
+          cache-from: type=gha
+          cache-to: type=gha,mode=max
```

Comment:
👀 oh, is this the trick?

Reply:
Yep, `mode=max` caches everything about PRs indefinitely across GitHub causing the whole service to come to a standstill.